src/common/popup/components/tool.jsx

```diff
@@ -4,25 +4,26 @@ import PropTypes from 'prop-types';
 import TicketList from './ticket-list';
 import NoTickets from './no-tickets';
 import Header from './header';
+
 import TicketShape from '../utils/ticket-shape';
+import ErrorShape from '../utils/error-shape';
 
-function Tool({ tickets }) {
-  const content = (tickets && tickets.length > 0)
+function Tool({ tickets, errors }) {
+  const content = tickets.length > 0
```

Comment:
Perfect! So we're only ever considering the errors if there are not tickets? 👍

Reply:
Exactly. Our idea from the last hacknight was that as long as a matching adapter finds tickets, it's okay for other adapters to reject with errors. So only if we don't receive any tickets, but we see errors, we report them.